docs/sphinx/developers/components.rst

```diff
@@ -90,9 +90,7 @@ not be any code here, just build system files.
 
 .. _forks-jai:
 
-:source:`forks/jai (JAI Image I/O Tools) <components/forks/jai>`:
-
-`Ant: jar-jai`
+`OME JAI (deprecated) <https://github.com/ome/ome-jai>`_:
```

Comment:
The OME codecs components should also be added to this page alongside the other upstream OME dependencies